docs/INSTALL-GUIDE-LINUX.md

```diff
@@ -0,0 +1,80 @@
+# Linux support
+
+This tutorial uses Ubuntu 20.04.
+
+Unless specified otherwise, all linux distributions are supported.
+
+## 1. install node.js and npm
+
+It is recommend to use [nvm (Node Version Manager)](https://github.com/nvm-sh/nvm#install--update-script),
+ use this link to install latest nvm version.
+
+For example:
+
+```Shell
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
```

Comment:
can this not be done with apt-get or something?

Reply:
no, apt only provides nodejs 10.19.xx (it was the old method by the way) default, and if we add the curl from nodejs repo we get package conflicts and broken dependencies

Reply:
i added a note for it to be clear